Story/RainMeadow.StoryHooks.cs

```diff
@@ -329,16 +329,10 @@ private void RainWorldGame_GoToDeathScreen(On.RainWorldGame.orig_GoToDeathScreen
             {
                 foreach (OnlinePlayer player in OnlineManager.players)
                 {
-                    if (OnlineManager.lobby.isOwner)
-                    {
-                        RPCs.GoToDeathScreen();
```

Comment:
This is an optimization

Reply:
```
                  if (player.id == OnlineManager.lobby.owner.id)
                    {
                        RPCs.GoToDeathScreen();
                    }
                    else
                    {

                        player.InvokeRPC(RPCs.GoToDeathScreen);
                    }
```

Keeps optimization and prevents unintended repeat invocations

Reply:
oh? What was causing the repeat invocations. There should have only ever been 1 lobby owner

Reply:
I believe it was from having a separate function called, "MovePlayersToDeathScreen" that simply iterated over all players and was called during this looping, so players were getting double-dosed with calls to move to sleep / death and dying of starvation rapidly. 

https://github.com/henpemaz/Rain-Meadow/pull/186/files

This solution should provide the same behavior while preventing repeats.

Reply:
yup checks out.